### DISCUSSION TITLE
What is the most spoken language in 🇮🇳?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#820] [Galaxy Trivia] [answered]
**Bonus:** What is the second popular?

![image](https://github.com/ParanoidUser/yolo/assets/5120290/52d4222f-aeaa-451b-9373-fabad2a6f9a6)

**Comment by ParanoidUser** [comment]
@DelrosarioRovic / @jomefavourite, any ideas? 😶‍🌫️

**Reply by jomefavourite** [reply]
Sadly I can't say because I don't live at 🇮🇳 and I'm also not Indian.

**Reply by DelrosarioRovic** [reply]
I think Hindi

**Comment by peterbaccaro** [comment]
As the world's oldest language, **Hindi** is the most spoken in India.  It is the first language of more than 40% of the Indian population. 

The second most spoken language in India is **Bengali**. It is primarily spoken in the eastern states of West Bengal, Tripura, and parts of Assam and Jharkhand. Bengali is the native language of the Bengali people and is spoken by a significant population in India, making it the second most widely spoken language in the country.

**Comment by SANKARAMDAS** [comment, marked as answer]
Hindi is the most widely spoken language with 602m Native speakers. According to the 2011 Indian Census, Hindi has 52.83 crore speakers and 43.63 per cent of the population speaking this language.
FunFact there is no National Language in India. 
The second most popular language is Bengali(also spelt as Bangla: বাংলা) with 272m Native Speakers, spoken by 8.03 per cent of the total population, is the sixth most spoken language in the world. 1999 UNESCO declared 21 Feb as the International Mother Language Day commemorating the Bengali Language Movement in 1952.
https://en.wikipedia.org/wiki/Languages_of_India

The image(above) is in Bengali. My native language is Bengali.
